ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/dorhook.js

```diff
@@ -89,12 +91,20 @@
             return false;
     }
 
+    function fetchAuthorContentFrameDocument() {
+        var contentFrameDocumentArray = ns.ContentFrame ? ns.ContentFrame.getDocument() : [];
+        if (contentFrameDocumentArray && contentFrameDocumentArray.length > 0) {
```

Comment:
not recommended `[0] `. should look for alternatives.

Reply:
ns.ContentFrame.getDocument() and document have two different values. 
We are using similar approach to get content frame container in cq-guides too: 

https://git.corp.adobe.com/livecycle/cq-guides/blob/master/content/src/main/resources/libs/fd/af/authoring/clientlibs/touchAuthoring/guideTouchAuthoringCommon/javascript/util/TouchUtils.js#L467

I'm not aware of another alternatives, can you please share the same.?